src/contractor.jl

```diff
@@ -70,6 +70,34 @@ end
 
 (C::Contractor{N,1,F1,F2})(A::Interval{T}, X::IntervalBox{N,T}) where {N,F1,F2,T} = C(IntervalBox(A), X)
```

Comment:
Maybe you need to add the `ex` type parameter in order to make this callable? (I'm not sure)